src/docs/user/references/ExecutionSchedulerServiceReference.adoc

```diff
@@ -7,56 +7,53 @@ The <<_glossary_execution_scheduler,*Execution Scheduler*>> schedules <<_glossar
 
 === Execution Calendars Object Syntax
 Below is an example <<_glossary_execution_calendars_object,*Execution Calendars Object*>>. It has a cron
-expression attached as "cron".
+expression attached as "cron" and a name as "name".
+The name must be unique and submission will be rejected if a name clashes with an already existing
+<<_glossary_execution_calendars_object,*Execution Calendars Object*>> inside the <<_glossary_execution_scheduler,*Execution Scheduler*>>.
 ----
 EXECUTION_CALENDARS =
 {
-    "cron": "0 23 * * MON-FRI",
+    "name": "weekdaysEveryHourAt23MinutsPast",
+    "cron": "0 23 * * MON-FRI"
 }
 ----
-One can see the cron expression "0 23 * * MON-FRI", which can thoroughly explained in the
-<<_execution_scheduler_cron_expression_syntax, Cron Expression Syntax>> section.
+One can see the cron expression "0 23 * * MON-FRI", which follows the quartz cron expression syntax explained in the
```

Comment:
here cron expression has to be at least 6 parts according to the quartz tutorials